packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -226,19 +229,28 @@ class InMemoryAnchorService implements AnchorService {
     return "<inmemory>"
   }
 
+  /**
+   * Stores the given AnchorServiceResponse as the current state of the anchor request to handle
+   * any future requests for the current status of the anchor, and then emits the response to
+   * the global event feed so that any currently polling subscriptions can receive it.
+   * @param asr
+   * @private
+   */
+  private _emitResponse(asr: AnchorServiceResponse) {
```

Comment:
@stbrody This is unnecessary. In the constructor you can add `this.#feed.subscribe(asr => this.#anchors.set(asr.cid.toString(), asr))`.

Reply:
Nice!